beginner/apps/src/bin/radio-recv.rs

```diff
@@ -17,6 +17,8 @@ fn main() -> ! {
     let mut radio = board.radio;
     let mut timer = board.timer;
 
+    radio.set_channel(Channel::_20);
```

Comment:
do we need a `// TODO change the channel value if you've changed the channel used by the dongle` comment here?

Reply:
a note would be good to have, yes "20 is the default channel of loopback.hex; you may need to change this if ran `change-channel`"

Reply:
There is a reminder in the text

Reply:
I added a reminder to the book

Reply:
yup, wasn't sure if we needed a "FYI this is the place that we meant" hint but you're right, it's pretty easy to spot